apps/chatter/serializers.py

```diff
@@ -1,12 +1,21 @@
 from rest_framework import serializers
 from rest_framework.renderers import JSONRenderer
+from django.contrib.auth import hashers
 
 from .models import Room, Entry
 
 class RoomSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Room
-        fields = ['id', 'name']
+        fields = ['id', 'name', 'password']
+
+    def create(self, validated_data):
+        if 'password' in validated_data:
+            password_hash = hashers.make_password(validated_data['password'])
+            validated_data['password'] = password_hash
+        else:
+            validated_data['password'] = ''
```

Comment:
If you had `null=True` on the password field, then you could skip having to set this to an empty string. Fore future reference.